dotcom-rendering/src/web/components/FixedSmallSlowIV.tsx

```diff
@@ -4,63 +4,72 @@ import { LI } from './Card/components/LI';
 import { UL } from './Card/components/UL';
 
 type Props = {
+	collectionId: string;
```

Comment:
What about `collectionId` being optional?

Reply:
It seems as though `showMore` and `collectionId` are coupled here. Is there a way to combine them into one property? Maybe we could use `showMoreId` and then if that exists we add the placeholder, and we can also use that when constructing the url? That way we no longer need the `showMore` boolean at all 🎉 

It's true that the value of the `showMoreId` would be the collection id, but I think it is valid for collection id to be _used_ as the value of another things id. This feels semantic to me

Reply:
I can see the appeal of this, but as far as I'm aware, the `collectionId` is a more general identifier. Even though I'm not sure if/where else we use it at the moment, it seems like the sort of id that might well be used in other contexts. Using its presence as an indicate of whether there are more cards to show would make that kind of re-use into a bigger refactor. Given that combining them would also introduce a new transformation between the Frontend model and the DCR model, I'm inclined to leave it as it is?

Reply:
I'd be happy to make both properties optional though (though it would be good if we had a way of ensuring that if 'hasMore' is set, then 'collectionId' should also be set.

Reply:
> might well be used in other contexts

This feels like we'd be 'coding for the future' which is famously impossible to do. You _think_ you know what future us will want but 99% of the time we get it very wrong!

> Using its presence as an indicate of whether there are more cards to show
I do agree that we shouldn't use `collectionId` as the indication for when to use show more. But if we call the prop `showMoreId` (or similar) then that is a lot clearer that this prop is only being used in this context.

Adding props has a cost and adding two props, which are coupled, is an even greater one.

> would also introduce a new transformation
It's true that this pattern would mean we would populate showMoreId using collectionId, but I think this is a valid thing to do. We routinely transform data in this way where the context demands it.

>  it would be good if we had a way of ensuring that if 'hasMore' is set, then 'collectionId' should also be set.
I feel that this is a good indication of why having two props is a problem 😉

Reply:
I've consolidated the two props in the new PR: https://github.com/guardian/dotcom-rendering/pull/5463